src/Symfony/Component/Console/Tests/Fixtures/application_2.txt

```diff
@@ -13,8 +13,9 @@ My Symfony application <info>v1.0</info>
   <info>-v|vv|vvv, --verbose</info>  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
 
 <comment>Available commands:</comment>
-  <info>help</info>                 Displays help for a command
-  <info>list</info>                 Lists commands
+  <info>help</info>                       Displays help for a command
```

Comment:
`getColumnWidth` considers aliases (`descriptor:alias_command3`), but we explicitly render those.. not sure how to solve it..